fastlane/Fastfile

```diff
@@ -17,4 +17,15 @@ platform :ios do
       test | xcpretty
       ")
   end
+
+  desc "Runs SwiftLint and returns any errors if our linter rule aren't met"
+  lane :lint do
+    swiftlint(
```

Comment:
Assuming we do not want warnings, would we also want the `strict` param? Assuming we want to have zero warnings in PRs